AndroidSDKCore/src/main/java/com/leanplum/internal/Util.java

```diff
@@ -660,7 +662,12 @@ public static void saveResponse(URLConnection op, OutputStream outputStream) thr
   private static String getResponse(HttpURLConnection op) throws IOException {
     InputStream inputStream;
     if (op.getResponseCode() < 400) {
-      inputStream = op.getInputStream();
+      String contentHeader = op.getHeaderField("content-encoding");
```

Comment:
Does `getHeaderField` trim white spaces too?

Reply:
"content-encoding" is the hardcoded key, we are looking for. Do you mean if the return is "g zip" or " gzip " as value? Does it handle that?
Can you confirm, is that what you mean, and do we need to check that?